### PR TITLE
 Add reference to az find from help

### PR DESCRIPTION
**Pull request by mirdaki**
This update informs users about the `az find` command when they use `-h` on a command with examples. The goal is to increase adoption of the command by making more people aware of it. It will print in a format like this:

```
Examples
    Redeploy a VM.
        az vm redeploy -g MyResourceGroup -n MyVm

    Redeploy all VMs in a resource group.
        az vm redeploy --ids $(az vm list -g MyResourceGroup --query "[].id" -o tsv)

For more specific examples, use: az find 'az vm redeploy'
```

As this solely modifies help, I haven't updated the HISTORY.rst file.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by mirdaki** [comment]
I've added a highlight to the suggestion as requested by @JasonRShaver

File: src/azure-cli-core/azure/cli/core/_help.py [hunk 1]
```diff
@@ -84,6 +85,9 @@ def _print_examples(help_file):
                 _print_indent(u'{0}'.format(e.long_summary), indent)
             _print_indent(u'{0}'.format(e.command), indent)
             print('')
+        indent = 0
+        message = 'For more specific examples, use: az find \'az {}\''.format(help_file.command)
```
Comment (yugangw-msft) [inline src/azure-cli-core/azure/cli/core/_help.py]: please use double quotes like `az find "az vm redeploy"`, because single quote won't work on Windows

**Comment by jiasli** [comment]
I would prefer to remove the color so that it is consistent with the previous output.

File: src/azure-cli-core/azure/cli/core/_help.py (hunk 1 above)
Comment (mirdaki) [reply src/azure-cli-core/azure/cli/core/_help.py]: OK I've updated it!

**Comment by JasonRShaver** [comment]
This change (to add highlighting) was made after a conversation with @achandmsft, I'll let you guys figure out what the best option is =)

**Comment by mirdaki** [comment]
Has there been any resolution on the text coloring?

**Pull request merged**